cfgov/unprocessed/css/apps/owning-a-home/tab.less

```diff
@@ -0,0 +1,101 @@
+
+.tabs,
+.tab-list {
+    margin: 0;
+    padding: 0;
+}
+
+.tabs {
+    margin-top: 2em;
+
+    button,
+    a {
+        position: relative;
+        display: block;
+        width: 100%;
+        padding: 28px 20px;
+        .u-webfont-regular();
+        font-size: 1.375em;
+        text-align: left;
+        z-index: 1;
+    }
+
+    button,
+    a,
+    a:link,
+    a:visited {
+        border-bottom: 1px solid @gray-10;
+        border-top: 4px solid @gray-10;
+        border-left: 0;
+        border-right: 0;
+        background: @gray-5;
+        color: @gray-80;
+    }
+
+    button:hover,
+    button:focus,
+    a:hover,
+    a:active,
+    a:focus {
+        color: @black;
+        background: #fff;
+        z-index: 2;
+    }
+
+    button:focus,
+    a:focus {
+        outline: 1px dotted @black;
+        outline-offset: 1px;
+    }
+
+    .active-tab {
+        button,
+        button:hover,
+        button:focus,
+        a,
+        a:link,
+        a:visited,
+        a:hover,
+        a:active,
+        a:focus {
+            border-bottom: none;
+            border-top: 4px solid @green-60;
+            background: #fff;
+            color: @black;
+        }
+    }
+}
+
+.tab-list {
+  list-style: none;
+}
+
+.tab-list + .tab-list {
+    button,
+    a {
+        border-left: 1px solid @gray-10;
+    }
+}
+
+.next-steps {
+  .tab-list {
+    .grid_column( 6 );
+    border-left: 0;
+    border-right: 0;
+  }
+}
+
+.tab-content {
+    margin-top: 3.125em;
```

Comment:
Should this em be calculated using `unit(…)` helper?

Reply:
Fixed.